### ISSUE TITLE
Vim hangs when trying any command

### ISSUE THREAD

**Issue by nilsi** [simplenote-vim/simplenote.vim#111]
I'm running Vim 8.1.2292 and installed the Simplenote plugin with Vundle.

When I run any command such as SimplenoteVersionInfo or SimplenoteList the tab is hanging and I cant do anything more than closing the tab.

I have my credentials in a separate file and source it in the top of .vimrc

`source ~/.simplenoterc`

I have been using Simplenote for a couple of years and have many notes in there but I guess SimplenoteVersionInfo should return directly. I tried waiting ~20min but nothing happens.

How can I debug this?

**Comment by atomicules** [comment]
Hmm... probably the best initial thing to do from a debugging point of view is try simplenote.py in a Python REPL and just see if you can authenticate and list notes, etc. Then depending on how that works out we can move back to Vim.

> I guess SimplenoteVersionInfo should return directly

It should in that if you try it on a blank buffer it should just say "This isn't a Simplenote".

Even if you had a stack of notes and it genuinely was taking a long time to list them all then `:SimplenoteNew` should work.

**Comment by nilsi** [comment]
I tried the python library by cd into `vim/bundle/simplenote.vim/autoload/simplenote.py` open up a python promp and then:

```
import simplenote

sn = simplenote.Simplenote('user', 'password')
sn.add_note('note')
```

It works fine. My current guess is that something is wrong in simplenote.vim

Maybe python does not work properly in vim or something:

I do not even seem to reach SimpleNoteNew function in the simplenote.vim script. I tried doing the following:

```
function! simplenote#SimplenoteNew()
echoerr "Do I reach here!?"
endfunction

```
But I got nothing, the prompt just hangs like before.

But If I put my error statement somewhere else in the file not in a function it errors out when I call the function.

**Comment by atomicules** [comment]
Odd. Fwiw, using vim 8.2.303 here compiled against Python 3.

Could you try running with a debug log and see if that shows anything interesting?

```
vim -V9debug.log
```

**Comment by nilsi** [comment]
I tried on a different Mac I got it working without problem. Comparing the vim installations by :version I could see that my computer that it worked on had +python3. The other computer had the default vim installed in osx. So I ran brew install vim and it works now.

Maybe I should have tried that earlier. I thought when it says +python for the vim version it would be using python2.7, maybe thats not the case.
